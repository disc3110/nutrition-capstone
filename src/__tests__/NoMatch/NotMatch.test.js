import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import {
  HashRouter as Router,
} from 'react-router-dom';
import NotMatch from '../../pages/NotMatch';
import store from '../../Redux/configureStore';

describe('Check if all components are rendered', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Provider store={store}><Router><NotMatch /></Router></Provider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
