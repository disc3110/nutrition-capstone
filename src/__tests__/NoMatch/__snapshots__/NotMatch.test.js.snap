// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if all components are rendered renders correctly 1`] = `
<div>
  <h3>
    No match for this page
  </h3>
</div>
`;
